src/lib/GOBLBuilder.svelte

```diff
@@ -30,7 +34,7 @@
         $editor = JSON.stringify(parsedValue.doc, null, 4);
         $envelope = parsedValue;
       } else {
-        $editor = JSON.stringify(parsedValue, null, 4);
+        $editor = data;
```

Comment:
This change is unrelated to the purpose/goal of this PR. It just ensures that we use the `data` from the parent component "as-is" when it's not wrapped in an envelope. This allows the parent project to set the editor with invalid/incomplete JSON in case it want to.